spec/views/collections/_sort_and_per_page.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'collections/_sort_and_per_page.html.erb', type: :view do
+  let(:blacklight_configuration_context) do
+    Blacklight::Configuration::Context.new(controller)
+  end
+  let(:search_state) { double('SearchState', to_h: {}, params_for_search: { sort: "score desc, system_create_dtsi desc" }) }
+  let(:user)  { stub_model(User, user_key: 'njaffer') }
+  let(:asset1)         { stub_model(GenericWork, id: "123abc", title: ["First of the Assets"], depositor: user.user_key) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(stub_model(User))
+    allow_any_instance_of(Ability).to receive(:can?).and_return(true)
+  end
+
+  it 'work is displayed' do
+    response_str = ActiveFedora::SolrService.instance.conn.get "select", params: { fq: ["id:\"#{asset1.id}\""], fl: ['id', Solrizer.solr_name(:collection)] }
+    expect(response_str["response"]["numFound"]).to eq 0
```

Comment:
ok yeah I see your point, the view is not even rendered here. I will update it. Thanks for your input!